src/main/resources/assets/notenoughupdates/gui/customtodos/overview.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Copyright (C) 2023 NotEnoughUpdates contributors
+  ~
+  ~ This file is part of NotEnoughUpdates.
+  ~
+  ~ NotEnoughUpdates is free software: you can redistribute it
+  ~ and/or modify it under the terms of the GNU Lesser General Public
+  ~ License as published by the Free Software Foundation, either
+  ~ version 3 of the License, or (at your option) any later version.
+  ~
+  ~ NotEnoughUpdates is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public License
+  ~ along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<Root xmlns="http://notenoughupdates.org/moulconfig">
+    <Gui>
+        <Column>
+            <Center>
+                <Scale scale="2">
+                    <Text text="Custom TODOs"/>
+                </Scale>
+            </Center>
+            <Spacer height="5"/>
+            <ScrollPanel width="350" height="200">
+                <Array data="@todos">
+                    <Row>
+                        <Text text="@label" width="200"/>
+                        <Switch value="@enabled"/>
+                        <Button onClick="@edit">
+                            <Text text="Edit"/>
+                        </Button>
+                        <Button onClick="@delete">
+                            <Text text="Delete"/>
+                        </Button>
+                    </Row>
+                </Array>
+            </ScrollPanel>
+            <Row>
+                <Button onClick="@addTodo">
+                    <Text text="Add new TODO"/>
+                </Button>
+                <Button onClick="@pasteTodo">
+                    <Text text="Paste TODO from Clipboard"/>
+                </Button>
+                <Spacer width="30"/>
+                <Button onClick="@save">
```

Comment:
I personally think it would be better to have a "Discard Changes" button and automatically save changes when closing the GUI.

afaIk all other NEU guis (exception being the item editor) work like this and I think it's what users would expect.